app/templates/components/addon-tabs.hbs

```diff
@@ -0,0 +1,63 @@
+<div class="addon-tabs">
+  <div class="addon-tabs--header">
```

Comment:
all of the buttons are missing the `aria-controls` attributes.  Please review https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role and rework this section.

Reply:
I have incorporated all of the items documented in that link into the new implementation 👍